src/kernel/panic.zig

```diff
@@ -217,6 +217,28 @@ fn parseNonWhitespace(ptr: [*]const u8, end: *const u8) PanicError![*]const u8 {
     return ptr + i;
 }
 
+///
+/// Parse until a newline character. Must be terminated by a newline character before the end
+/// address.
+///
+/// Arguments:
+///     IN ptr: [*]const u8 - The address at which to start looking.
+///     IN end: *const u8 - The end address at which to start looking. A whitespace character must
```

Comment:
Do you mean a new-line character rather than a whitespace character?

Reply:
yep, missed one :P